### PR TITLE
Fix for scrolling

### PR DESCRIPTION
**Pull request by ramramps**
This PR fixes the scrolling in LibrarieJS, after the category is clicked in search.

giphy 1

### CODE REVIEW

File: src/components/LibraryItem.tsx [hunk 1]
```diff
@@ -111,6 +111,17 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
         }
     }
 
+    //Afer rendering each Library item, scroll to the expanded item
+    componentDidMount() {
+        if (this.props.data.expanded) {
+            let test = this.props.data;
```
Comment (ramramps) [inline src/components/LibraryItem.tsx]: still testing this. found few edge cases. updating this code.

**Comment by mjkkirschner** [comment]
@ramramps can you add a gif of this behavior? Also, how does this interact with - https://github.com/DynamoDS/librarie.js/pull/102

**Comment by ramramps** [comment]
I will add a gif. This doesn't change the other scroll behavior.

**Comment by ramramps** [comment]
ping @mjkkirschner. GIF added.

**Comment by Racel** [comment]
Looks great, @ramramps!

**Comment by mjkkirschner** [comment]
LGTM @ramramps !
